introduction.md

```diff
@@ -3,3 +3,4 @@ Favorite Code School path is Javascript
 List of deadly skills: 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?